draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -717,6 +774,175 @@ Steps:
 23. Output (x, y)
 ~~~
 
+### Boneh-Franklin Method {#supersingular}
+
+The map2curve_bf(alpha) implements the Boneh-Franklin method {{BF01}} which
+covers the case of supersingular curves `E: y^2=x^3+B`.
+
+**Preconditions**
+
+This algorithm works for any Weierstrass curve over `F_q` such that `A=0` and
+`q=2 mod 3`.
+
+**Examples**
+
+- `y^2=x^3+1`
+
+**Algorithm**: map2curve_bf
+
+Input:
+
+ - `alpha`: an octet string to be hashed.
+ - `B`: the constant from the Weierstrass curve.
+
+Output:
+
+ - `(x,y)`: a point in E.
+
+Operations:
+
+~~~
+1. u = HashToBase(alpha)
+2. x = (u^2-B)^(1/3)
```

Comment:
Should we express this cube root as `(u^2 - B)^((2*q - 1)/3)`?

Reply:
Fixed.